android-pdf-viewer/src/main/java/com/github/barteksc/pdfviewer/PDFView.java

```diff
@@ -1607,8 +1608,12 @@ public Configurator onLongPress(OnLongPressListener onLongPressListener) {
             return this;
         }
 
-        public Configurator linkHandler(LinkHandler linkHandler) {
-            this.linkHandler = linkHandler;
+        public Configurator tapHandler(Uri uri, OnAnnotationPressListener listener) {
+            this.tapHandler = new DefaultTapHandler(
+                    PDFView.this,
+                    uri,
+                    listener
+            );

```

Comment:
Changing this to 

```kotlin
public Configurator onTap(Uri uri, OnAnnotationPressListener listener) {
            this.onTapListener = new DefaultTapHandler(PDFView.this, uri, listener);
            return this;
        }
```

is the way to re use the existing tap listener

Reply:
Deleted some unneeded code and this still works as expected